### DISCUSSION TITLE
Streaming data transfer

### DISCUSSION THREAD

**Discussion by MRtecno98** [redis/go-redis#2478] [Q&A]
Can i use this library to get a binary blob from redis as a byte stream(like with an `io.Reader`) and vice-versa?

**Comment by vmihailenco** [comment]
From what I know Redis protocol does not support streaming.

**Reply by MRtecno98** [reply]
Well i mean it's still a socket in the end right? It's not really a protocol feature being able to stream data out of the socket or is it?

**Reply by vmihailenco** [reply]
You probably could try to obtain `*redis.Conn` and then retrieve the underlying socket to write/read whatever you want. Retrieve the underlying socket is not currently supported.
